### PR TITLE
added conda installation in docs

### PR DESCRIPTION
**Pull request by satyam-jha**
i have added conda installation in docs

### CODE REVIEW

**Review by mfeurer — COMMENTED** [review]
Thanks a lot for your contribution @satyam-jha. However, the docs now contradict themselves. There's now a section "Anaconda installation" that says that Auto-sklearn cannot be installed via Anaconda, and a section which describes how to do this. Could use please make the section "Anaconda installation" two section, one about installing the package via conda-forge, and one in the spirit of the current description? Also, could you please give a link to conda forge docs for people who don't know it? Thanks a lot!

**Comment by satyam-jha** [comment]
i have added "conda forge" section inside the "Anaconda installation" section and added links for both conda forge docs and feedstocks repos link

File: doc/installation.rst [hunk 1]
```diff
@@ -70,6 +70,38 @@ be solved by installing the *gcc* compiler shipped with AnaConda (as well as
...
+Installing `auto-sklearn` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+
+.. code:: bash
+
+    conda config --add channels conda-forge
+
+    conda config --set channel_priority strict
+
+
+Once the `conda-forge` channel has been enabled, `auto-sklearn` can be installed with:
```
Comment (franchuterivera) [inline doc/installation.rst:85]:
```suggestion
You must have `conda >=4.9`. To update conda or check your current conda version, please follow the instructions from `the official anaconda documentation <https://docs.anaconda.com/anaconda/install/update-version/>`_ . Once the `conda-forge` channel has been enabled, `auto-sklearn` can be installed with:
```

**Comment by franchuterivera** [comment]
Hello, thanks a lot for this PR. I had some suggestions to make the installation instructions a bit more clear.

I have added some commit suggestion because the instructions did not fully work for me. The problem is that I had conda 4.8. 

Additionally, could you please also update the statement in this file about:

```
Anaconda does not ship *auto-sklearn*, and there are no conda packages for*auto-sklearn*. Thus, it is easiest to install *auto-sklearn* as detailed inthe Section `Installing auto-sklearn`_.
```

Maybe to something like:
```
You need to enable conda-forge to install *auto-sklearn* via anaconda. This section explains how to enable conda-forge, to be able to install *auto-sklearn* with `conda install auto-sklearn`. Optionally, you can also install *auto-sklearn* with `pip` as detailed in the Section `Installing auto-sklearn`_. 
```

**Pull request merged**